src/views/v2/widgets/events-list/event/title.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Widget: Events List Event Title
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/v2/widgets/events-list/event/title.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link http://m.tri.be/1aiy
+ *
+ * @version TBD
+ *
+ * @var WP_Post $event The event post object with properties added by the `tribe_get_event` function.
+ *
+ * @see tribe_get_event() For the format of the event object.
+ */
+?>
+<h3 class="tribe-events-widget-events-list__event-title tribe-common-h7">
+	<a
+		href="<?php echo esc_url( $event->permalink ); ?>"
+		title="<?php echo esc_attr( $event->title ); ?>"
+		rel="bookmark"
+		class="tribe-events-widget-events-list__event-title-link tribe-common-anchor-thin"
+	>
+		<?php
+		// phpcs:ignore
+		echo $event->title;
```

Comment:
Should this be escaped? The code that produces that string is *not* escaping it: https://github.com/moderntribe/the-events-calendar/blob/5fc1b024de10e801f7799e250470f6035b97b273/src/Tribe/Models/Post_Types/Event.php#L237

Reply:
I believe titles can contain HTML, which is why we don't escape it. This is how it is in the views as well.

Reply:
Late to the party, but `wp_kses_post` might be a reasonable alternative?